frontend/src/js/external-forms/form-concept-group/FormConceptGroup.tsx

```diff
@@ -199,6 +208,53 @@ const FormConceptGroup = (props: Props) => {
             ? t("externalForms.common.concept.copying")
             : props.attributeDropzoneText
         }
+        dropBetween={(i: number) => {
+          return (item: PossibleDroppableObject) => {
+            if (item.type !== DNDType.CONCEPT_TREE_NODE) return;
+
+            if (props.isValidConcept && !props.isValidConcept(item))
+              return null;
```

Comment:
Feels inconsistent to return undefined above and null below, could you make this consistent?